### PR TITLE
[sc-164648] update more repo type info with gitlab

### PR DESCRIPTION
**Pull request by jazanne**
- circle ci orb
- alphabetize repo type options

### CODE REVIEW

File: docs/CONFIGURATION.md [hunk 1]
```diff
@@ -63,9 +63,9 @@ Flags:
 
   -r, --repoName string            Repository name. Will be displayed in LaunchDarkly. Case insensitive. Repo names must only contain letters, numbers, '.', '_' or '-'."
 
-  -T, --repoType string            The repo service provider. Used to correctly categorize repositories in the LaunchDarkly UI. Aceptable values: github|bitbucket|gitlab|custom. (default "custom")
+  -T, --repoType string            The repo service provider. Used to correctly categorize repositories in the LaunchDarkly UI. Aceptable values: bitbucket|custom|github|gitlab. (default "custom")
 
-  -u, --repoUrl string             The display url for the repository. If provided for a github or bitbucket repository, LaunchDarkly will attempt to automatically generate source code links.
+  -u, --repoUrl string             The display url for the repository. If provided for a bitbucket, github, or gitlab repository, LaunchDarkly will attempt to automatically generate source code links.
```
Comment (tiffanylphan) [inline docs/CONFIGURATION.md:68]: It would be good to use correct capitalization for Bitbucket, GitHub, GitLab

Comment (jazanne) [reply docs/CONFIGURATION.md:68]: hm, i would almost rather it say this instead

https://github.com/launchdarkly/ld-find-code-refs/blob/15723036994b6eac14506754283eae9c4d4126cf/options/flags.go#L43-L45

**Review by ember-stevens — APPROVED** [review]

**Comment by jazanne** [comment]
> I noticed in some places the code snippet, eg: `repoType` is used. In other places cone snippets are not used while referring to the same thing. We should have consistency.

looks like [nowhere that the backticks are used are they actually being displayed](https://circleci.com/developer/orbs/orb/launchdarkly/ld-find-code-refs). just going to remove them

**Review by mmrj — APPROVED** [review]

**Comment by jazanne** [comment]
> > I noticed in some places the code snippet, eg: `repoType` is used. In other places cone snippets are not used while referring to the same thing. We should have consistency.
> 
> looks like [nowhere that the backticks are used are they actually being displayed](https://circleci.com/developer/orbs/orb/launchdarkly/ld-find-code-refs). just going to remove them

nevermind, backticks are used in circleci orb because of yaml syntax with double quotes. not going to modify that file, but updated the flag opts to wrap names in double-quotes, which aligns to go file formatting

**Review by tiffanylphan — APPROVED** [review]

**Pull request merged**